static/js/compose.js

```diff
@@ -1134,7 +1134,7 @@ $(function () {
             break;
         case 'FileTooLarge':
             // sanitization not needed as the file name is not potentially parsed as HTML, etc.
-            msg = "\"" + file.name + "\"" + i18n.t(" was too large; the maximum file size is 25MiB.");
+            msg = "\"" + file.name + "\" " + i18n.t("was too large; the maximum file size is 25MiB.");
```

Comment:
I agree with you about that being more readable, but it's also more risky because it unnecessarily adds a variable in the strings to translate.

Not sure if it's worth, what do you think?